<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>350</width>
    <height>195</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>350</width>
    <height>195</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>350</width>
    <height>195</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Data Node</string>
  </property>
  <widget class="QLabel" name="label_1">
   <property name="geometry">
    <rect>
     <x>5</x>
     <y>5</y>
     <width>75</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Level:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>5</x>
     <y>35</y>
     <width>75</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Type</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>5</x>
     <y>65</y>
     <width>75</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Tag:</string>
   </property>
  </widget>
  <widget class="QComboBox" name="levelCB">
   <property name="geometry">
    <rect>
     <x>85</x>
     <y>5</y>
     <width>260</width>
     <height>30</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="typeCB">
   <property name="geometry">
    <rect>
     <x>85</x>
     <y>35</y>
     <width>260</width>
     <height>30</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>Text</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Media</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Table</string>
    </property>
   </item>
  </widget>
  <widget class="QLineEdit" name="tagLE">
   <property name="geometry">
    <rect>
     <x>85</x>
     <y>65</y>
     <width>260</width>
     <height>30</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="browseL">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>130</y>
     <width>150</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="browsePB">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>130</y>
     <width>95</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>Browse</string>
   </property>
  </widget>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>179</x>
     <y>165</y>
     <width>166</width>
     <height>25</height>
    </rect>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QRadioButton" name="allRB">
   <property name="geometry">
    <rect>
     <x>5</x>
     <y>100</y>
     <width>170</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>All (Separator: /@#$/)</string>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QRadioButton" name="singleRB">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>100</y>
     <width>40</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>At</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="positionSB">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>100</y>
     <width>40</width>
     <height>25</height>
    </rect>
   </property>
   <property name="minimum">
    <number>1</number>
   </property>
   <property name="maximum">
    <number>100</number>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>265</x>
     <y>100</y>
     <width>80</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>th position</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>singleRB</sender>
   <signal>toggled(bool)</signal>
   <receiver>positionSB</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>99</x>
     <y>112</y>
    </hint>
    <hint type="destinationlabel">
     <x>139</x>
     <y>112</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
